crux-core/src/crux/api/alpha/NodeStatus.java

```diff
@@ -0,0 +1,47 @@
+package crux.api.alpha;
+
+import clojure.lang.Keyword;
+
+import java.util.Map;
+
+public class NodeStatus {
+    public final String version;
+    public final String revision;
+    public final String kvStore;
+    public final int estimatedNumberOfKeys;
+    public final long size;
+    public final long indexVersion;
+
+    @Override
+    public String toString() {
+        return "NodeStatus{" +
```

Comment:
IntelliJ'll generate you a String.format based toString if you ask it to, I think? preferable to string concatenation for both performance and (more subjective) readability

Reply:
Might need to check that? I think this is what it does by default, so will need to have a look at my options